README.rst

```diff
@@ -192,17 +195,36 @@ configure the settings associated with the target storage type.
 |WRITE_AWS_S3_HOST           | http://localhost:8000 | URL of S3               |
 +----------------------------+-----------------------+-------------------------+
 
+
 Running the application
 -----------------------
 
 To run the application from the command line:
 
+For pip installed versions:
+
   ``data-transfer``
 
+Calling the applicaiton directly:
+
+  ``python bin/data-transfer``
 
 For production use we recommend running the application using PM2:
 
-  ``pm2 start bin/data-transfer --name data-transfer --interpreter python``
+  ``pm2 start ecosystem.config.js --only data-transfer``
```

Comment:
This fails if `pm2` isn't already installed globally. We should add a note or instructions about running `npm install pm2 -g ` prior to this step (if it is not already installed globally).